### PR TITLE
16 pathology tumor detection

### LINKED ISSUE #16: Pathology Tumor Detection Bundle
Add a MONAI Bundle for pathology tumor detection pipeline.

### PR DESCRIPTION
**Pull request by bhashemian**
Fixes #16 

### Description
MONAI Bundle for pathology metastasis detection pipeline

### Status
**Ready**

### Please ensure all the checkboxes:

- [x] Codeformat tests passed locally by running `./runtests.sh --codeformat`.
- [ ] In-line docstrings updated.
- [x] Update `version` and `changelog` in `metadata.json`.
- [x] Ensure versions of packages such as `monai`, `pytorch` and `numpy` are correct in `metadata.json`.
- [x] Descriptions should be consistent with the content, such as `eval_metrics` of the provided weights and TorchScript modules.
- [x] Files larger than 25MB are excluded and replaced by providing download links in `large_file.yml`.

### CODE REVIEW

File: models/pathology_tumor_detection/large_file.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+large_files:
```
Comment (yiheng-wang-nv) [inline models/pathology_tumor_detection/large_file.yml]: Hi @drbeh , could you please help to modify the `large_files.yml` ? Thanks!

**Comment by yiheng-wang-nv** [comment]
Hi @drbeh , it seems the md5 value of larges files are wrong (https://github.com/Project-MONAI/model-zoo/runs/7581568864?check_suite_focus=true#step:2:167) could you please double confirm it? Thanks!

**Comment by bhashemian** [comment]
Hi @yiheng-wang-nv, I checked the hash values and they were fine. I believe the cause of the error is that it cannot download from google drive using the web link url. I changed the google drive url prefix and it should work now. Please let me know if the error persists. Thanks

File: models/pathology_tumor_detection/configs/metadata.json [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+                    255
+                ],
+                "is_patch_data": true,
+                "channel_def": {
+                    "0": "image"
+                }
+            }
+        },
+        "outputs": {
+            "pred": {
```
Comment (yiheng-wang-nv) [inline models/pathology_tumor_detection/configs/metadata.json:54]: Hi @drbeh , according to the schema: https://github.com/Project-MONAI/MONAI-extra-test-data/releases/download/0.8.1/meta_schema_20220324.json
for the output, the following properties are required:
```
type
format
num_channels
spatial_shape
dtype
value_range
```
Could you please help to add the missing parts? Thanks!

Comment (bhashemian) [reply models/pathology_tumor_detection/configs/metadata.json:54]: Thank you @yiheng-wang-nv! I updated metadata.json and checked the schema validation, so it should be fine now.

**Comment by bhashemian** [comment]
Hi @yiheng-wang-nv, `torchvision` dependency was missing from metadata.

**Comment by yiheng-wang-nv** [comment]
> Hi @yiheng-wang-nv, `torchvision` dependency was missing from metadata.

Hi @drbeh , it seems the torchvision version has conflicts with torch version. Could you double check which pytorch version you used? Thanks!

**Comment by bhashemian** [comment]
> Hi @drbeh , it seems the torchvision version has conflicts with torch version. Could you double check which pytorch version you used? Thanks!

Hi @yiheng-wang-nv, I updated the package versions. Could you please run ci tests again? Thanks

File: models/pathology_tumor_detection/configs/metadata.json [hunk 3]
```diff
@@ -5,11 +5,11 @@
         "0.1.0": "initialize release of the bundle"
     },
     "monai_version": "0.9.1",
-    "pytorch_version": "1.10.0",
+    "pytorch_version": "1.12.0",
     "numpy_version": "1.21.2",
+    "torchvision": "0.13.0",
+    "cucim": "22.04",
     "optional_packages_version": {
-        "cucim": "22.04",
```
Comment (yiheng-wang-nv) [inline models/pathology_tumor_detection/configs/metadata.json]: HI @drbeh , here could you put torchvision and cucim back into `optional_packages_version`?

Comment (bhashemian) [reply models/pathology_tumor_detection/configs/metadata.json]: Hi @yiheng-wang-nv, sure, but can let me know why is that? I don't get the concept of optional here as they are required for this bundle.

Comment (Nic-Ma) [reply models/pathology_tumor_detection/configs/metadata.json]: Hi @drbeh ,

I think we treat cucim and torchvision as optional dependencies of MONAI:
https://docs.monai.io/en/latest/installation.html#installing-the-recommended-dependencies
So here the "optional" means the same.

Thanks.

**Comment by yiheng-wang-nv** [comment]
Hi @drbeh , `monai.data.dataset.CSVDataset` requires an optional lib `pandas`, could you please also include it into the metadata? Thanks

**Review by yiheng-wang-nv — APPROVED** [review]
Thanks @drbeh , this PR looks good!

**Review by Nic-Ma — APPROVED** [review]

**Pull request merged**